recipe/meta.yaml

```
@@ -1,21 +1,21 @@
 {% set name = "numba" %}
-{% set version = "0.60.0" %}
+{% set version = "0.61.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 5df6158e5584eece5fc83294b949fd30b9f1125df7708862205217e068aabf16
+  sha256: 888d2e89b8160899e19591467e8fdd4970e07606e1fbc248f239c89818d5f925
   patches:
     # This forces a benign DeprecationWarning in test_import.py to be ignored.
     - patches/ignore_deprecation_warning_in_import_test.patch  # [py>=310 and win]
 
 build:
   number: 1
   skip: true  # [python_impl == 'pypy']
-  skip: true  # [(py<39 or py>=313) or s390x]
+  skip: true  # [py<310 or s390x]
```

```suggestion
  skip: true  # [(py<310 or py>=314) or s390x]
```
https://github.com/numba/numba/blob/0.61.0/setup.py#L23